serverless-sample/app/gcs/Makefile

```diff
@@ -0,0 +1,6 @@
+
+build:	
+	CGO_ENABLED=0 GOOS=linux go build -v -installsuffix nocgo -o .builds/binary-gcp main.go
+upload:	
+	docker build -t gcr.io/<project-name>/cloudrun-server:latest .
```

Comment:
so I didn't clue in there were placeholders here. Might be better to use an env var that could be set by the user so that one doesn't have to edit this file. Usage could be doc'd in the readme...something like this:
$ GCP_PROJECT_NAME=myproj make upload
(could do similar for aws)

Also, I would recommend adding a clean step to delete the binary and have a make dependency on the upload:
upload: build
            docker build....

Reply:
fixed